data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,401 @@ mission "Remnant: Face to Maw 2B"
+
+
+
+mission "Remnant: Crystal Research 6"
+	name "Scan the <npc> in <waypoints>"
+	description `Go scan the <npc> to record the Key Stone post-wormhole.`
+	landing
+	waypoint "Peragenor"
+	source "Viminal"
+	destination "Viminal"
+	npc "scan outfits" "save"
+		government "Remnant"
+		personality coward disables plunders staying surveillance target mining
+		system "Peragenor"
+		ship "Starling" "Winds of Light"
+	to offer
+		has "Remnant: Crystal Research 5: done"
+	on offer
+		conversation
+			`	"Ready? Let's go!" Aeolus signs as he fires up his systems and gestures an acknowledgement to the dockworker. The worker nimbly steps clear and watches as the two ships head back out of the dome.`
+				launch
+	on visit
+		dialog `You land on <planet>, but you haven't scanned the <npc> in <waypoints>. Make sure you scan the ship before returning.`
+	on complete
+		payment 150000
+		conversation
+			`The scanning proceeds smoothly, and while it does note some differences between the two crystals, they both appear to function similarly. After you return to Viminal, Aeolus comes over to collect a copy of the data for the researchers.`
+			`	"Thanks for your help, Captain <first>. This information may be valuable in understanding how the wormholes work." He pauses, then continues thoughtfully: "Perhaps more importantly, it may help us understand how the wormholes of the Ember Waste are different from the wormholes elsewhere that require no such aid."`
+			`	He looks down at the data crystal. "I have one more task to ask of you. Meet me in the cafeteria in an hour to discuss it. May the Embers burn bright for you, Captain." He gives a wave that seems vaguely like a salute and heads off.`
+			`	As he disappears from sight you hear a faint chime from your commlink, which displays a message indicating that two payments of <payment> have been allocated to your account. You recall that you didn't get a message after the last mission, so they must have combined them. You do find it odd, however, that they always seem to know exactly when you finish a delivery.`
+
+
+
+mission "Remnant: Crystal Research 7"
+	name "Deliver Data to <destination>"
+	description `A copy of the scanner data needs to be delivered to the researchers on <destination>.`
+	source "Viminal"
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Crystal Research 6: done"
+	on offer
+		conversation
+			`You walk into the cafeteria and spot Aeolus halfway through an impressively large meal. He waves you over, and by the time you are seated a tray with a significantly smaller meal is waiting for you. While you have acclimatized to a few of the dishes, most of them are still very unappetizing.`
+			choice
```

Comment:
Maybe add a choice here that doesn't involve somehow commenting on the food.

Reply:
I have now added a "Straight to business" option to the conversation. It isn't nearly as interesting as the other options, but it is a valid option that I'm sure some players will prefer to take.